src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js

```diff
@@ -37,7 +37,16 @@ define(function () {
                 toolbar: {
                     buttons: {
                         add: {},
-                        deleteSelected: {}
+                        deleteSelected: {},
+                        export: {
+                            options: {
+                                urlParameter: {
+                                    flat: true,
+                                    limit: 999999
```

Comment:
This is somehow related to https://github.com/sulu-io/sulu/issues/1011, but on the other hand it's also completely different :smile: 

Actually we said once that there should be always a limit on the REST API, maybe we have to rethink that under this circumstances. Or we can omit this one here, and use null as default limit only for the csv format? /cc @chirimoya